terragrunt/components/terragrunt.hcl

```diff
@@ -268,7 +268,7 @@ locals {
   }
 
   # @TODO (ABN) Remove me
-  desired_count_development    = 8
+  desired_count_development    = 1
```

Comment:
Does the comment need to go as well?

Reply:
No, the comment should remain for the entire section. 
Once we’re done testing the shared session (and now the database connections as well), we’ll revert back to having all pre-production accounts use the same configuration.